automata/fa/dfa.py

```diff
@@ -148,6 +150,23 @@ def __invert__(self):
         """Return the complement of this DFA and another DFA."""
         return self.complement()
 
+    def __contains__(self, other):
```

Comment:
@Tagl Could you please add docstrings to this and the other magic methods below?